packaging/gunicorn/ansible-runner-service.spec

```diff
@@ -88,11 +88,15 @@ install -m 644 ./wsgi.py %{buildroot}%{_sysconfdir}/ansible-runner-service
 mkdir -p %{buildroot}%{_unitdir}
 cp -r ./packaging/gunicorn/ansible-runner-service.service %{buildroot}%{_unitdir}
 
+mkdir -p %{buildroot}/var/log/ovirt-engine
+touch %{buildroot}/var/log/ovirt-engine/ansible-runner-service.log
```

Comment:
Isn't /var/log/ovirt-engine directory enough? Do we need also to add empty file there?

Reply:
Also as this is dev env, we don't have logrotate rule included as a part of engine installation, so we should probably add logrotate rule directly into ansible-runner-service-dev package

Reply:
We have the logrotate rule in engine https://gerrit.ovirt.org/#/c/109039/10/packaging/sys-etc/logrotate.d/ovirt-engine.in
And the  /var/log/ovirt-engine directory might be enough I have not tested it without and just wanted to be safe.

Reply:
It doesn't make sense for dev env, because in dev env everything is installed under <PREFIX> directly, to logrotate files fill be under <PREFIX>/etc/logrotate.d so completely unusable.

So maybe it's worth to have logrotate for dev env ...

Reply:
You are right my mistake, added the logrotate.